site/_includes/holder.svg

```diff
@@ -11,13 +11,11 @@ optional: text_size - "13" (default)
 {%- assign width = size[0] | default: "800" -%}
 {%- assign height = size[1] | default: "400" -%}
 
-{%- capture text -%}
-  {%- if include.text == null -%}
-    {{ width }}×{{ height }}
-  {%- else -%}
-    {{ include.text }}
-  {%- endif -%}
-{%- endcapture -%}
+{%- if include.text == null -%}
```

Comment:
`null` or `nil`?

Reply:
Also, you could probably use `default` here

Reply:
I've kept the condition consistent with the ones from [`example.html`](https://github.com/twbs/bootstrap/blob/v4-dev/site/_includes/example.html)

Reply:
I'll have a look myself later because `default` is cleaner IMO.

Reply:
Actually we can't do that here, so your solution is fine 👍